openedx/stanford/lms/envs/common.py

```diff
@@ -115,6 +115,7 @@
 PROGRESS_SUCCESS_BUTTON_URL = 'http://<domain>/<path>/{course_id}'
 PROGRESS_SUCCESS_BUTTON_TEXT_OVERRIDE = None
 SEARCH_FILTER_GENERATOR = 'openedx.stanford.lms.lib.courseware_search.lms_filter_generator.TileSearchFilterGenerator'
+SEARCH_DISCOVERY_SORT_FIELDS = '_score:desc,start:desc'
```

Comment:
What kind of values do we see for `_score`?
- integers?
- what kind of range?
- what's the median?
- etc..

Reply:
@stvstnfrd It's a positive floating point number. Nitty gritty explained here https://www.compose.com/articles/how-scoring-works-in-elasticsearch/